.github/workflows/tests.yml

```diff
@@ -81,3 +81,27 @@ jobs:
         env:
           BUILD_ID: ${{ github.run_id }}
           BUILD_ENV: GitHub Actions
+
+  cypress:
+    runs-on: ubuntu-latest
+    container:
+      image: saucelabs/stt-cypress-mocha-node:v0.1.3
+
+    steps:
+      # appears that checkout@v2 uses javascript which is not compatible 
+      # with the included node version in the container image.
+      - name: Checkout Code
+        uses: actions/checkout@v1
+        
+      - name: Workaround for container permissions
+        run: sudo chown -R $USER:$(id -gn $USER) /github/home
+      
+      - name: Workaround for container folder
+        run: sudo ln -s /home/seluser/.cache /github/home/.cache
```

Comment:
Do we have an internal ticket to follow up on these workarounds? It would be great if we can get rid of them as they add complexity to the workflow

Reply:
Done. Ticket created.